pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Environment.groovy

```diff
@@ -266,4 +253,11 @@ public class Environment implements Serializable {
         public boolean isLiteral
         public Object value
     }
+
+    private String resolveAsScript(Binding binding, String script) {
+        String toRun = Utils.prepareForEvalToString(script)
+        SecureGroovyScript toExec = new SecureGroovyScript(toRun, true)
+            .configuring(ApprovalContext.create().withCurrentUser().withKey(Utils.getCurrentJobName()))
```

Comment:
Hmm, actually, https://github.com/jenkinsci/script-security-plugin/blob/master/src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ApprovalContext.java#L104 says that's only relevant for whole script approvals...since we're sandboxing the `SecureGroovyScript`, we are doing signature approvals, not whole script approvals, so this should be ok.